lib/python/src/aria/ops/definition/parameter.py

```diff
@@ -147,14 +148,25 @@ def __init__(
             raise DuplicateKeyException(
                 f"Duplicate enum value in parameter {key}: {values}."
             )
+
         self.values = values
-        if default not in self.values and default is not None:
-            self.values.append(default)
+        if (
+            default not in [v[0] if isinstance(v, tuple) else v for v in self.values]
+            and default is not None
+        ):
+            self.values.append((default, default))
 
     def to_json(self) -> dict:
         return super().to_json() | {
-            "type": "string",
+            "type": "dict",
```

Comment:
The 'type' here is what type of value the identifier holds, not the type of the json. It should remain 'string', because enums return a string rather than a numeric value